packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2507,6 +2534,7 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
       return new _CleanSemanticsFragment(renderObjectOwner: this);
     }
     List<_SemanticsFragment> children;
+    bool dropSemanticsOfLeftSiblings = (this is RenderBlockSemantics);
```

Comment:
I refactored this to use the same mechanism we use for `isSemanticBoundary`. No more testing `this` against a specific subclass and no more cyclic imports!